pkg/git/repository.go

```diff
@@ -160,7 +159,7 @@ func (o *CloneOptions) URL() string {
 }
 
 func (o *CloneOptions) Revision() string {
-	return plumbing.ReferenceName(o.revision).Short()
+	return o.revision
```

Comment:
do we even need this?

Reply:
Yeah, when creating an Argo CD Application, the Source.Revision field will contain this value.